packages/core/src/fhir-deduplication/__tests__/examples/immunization-examples.ts

```diff
@@ -8,3 +8,14 @@ export const cvxCodeFlu = {
   ],
   text: "Influenza, split virus, quadrivalent, PF",
 };
+
+export const ndcCodeFlu = {
```

Comment:
I suggest we create functions for these, setting the return type to help read it. It doesn't add any overhead to code, and it simplifies to extend it to send params to make the result more dynamic.

For example: when I read [this](https://github.com/metriport/metriport/pull/3106/files#diff-244c0570c3878051da5da52b6880218492eb142d2b9c27813510a95b1f3b7f38R252-R253) I have to wonder where is this `dateTime` object coming from, what's the value of the `start` property.

```ts
    observation.effectiveDateTime = dateTime.start;
    observation2.effectiveDateTime = dateTime.start;
```

But when I read the code below, I know its a newly created Period just for this test - not being reused or with any particular meaning.

```ts
    const period = makePeriod();
    observation.effectiveDateTime = period.start;
    observation2.effectiveDateTime = period.start;
```